sdk/servicebus/azure-servicebus/tests/perf_tests/T1_legacy_tests/receive_message_batch.py

```diff
@@ -0,0 +1,21 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import asyncio
+
+from ._test_base import _ReceiveTest
+
+
+class LegacyReceiveMessageBatchTest(_ReceiveTest):
+    def run_sync(self):
+        batch = self.receiver.fetch_next(max_batch_size=self.args.num_messages)
+        if self.args.peeklock:
+            for msg in batch:
+                msg.complete()
```

Comment:
if `num_messages` is very large, then we are not able to receive all the messages in a single call.

should it be like the push mode style?
```python
count = 0
while count < self.args.num_messages:
     batch = self.receiver.fetch_next(max_batch_size=self.args.num_messages)
     if self.args.peeklock:
            for msg in batch:
                msg.complete()
     count += len(batch)
```

Reply:
Yeah good plan. Updated!